docs/admin/setup/distributed.rst

```diff
@@ -504,6 +504,118 @@ It relays message between instances:
+
+The run configuration contains just two profiles:
+
+::
+
+    --spring.profiles.active:prod,buildagent
+
+Build agents depend on much less services than the core nodes, thus we can adapt the ``application-prod.yml`` to exclude some of these dependencies.
+This heavily reduces the application start up time and resource demand.
+
+You can make following adaptations to the ``application-prod.yml``:
+
+1. Disable Liquibase and loadbalancer cache:
+
+.. code-block:: yaml
+
+    spring:
+        liquibase:
+            enabled: false
+        cloud:
+            loadbalancer:
+                cache:
+                    enabled: false
+
+2. Autoconfigure exclusions
+
+.. code-block:: yaml
+
+    spring:
+        autoconfigure:
+            exclude:
+                # Hibernate and DataSource are not needed in the build agent
+                - org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration
+                - org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
+                # Those metrics are repeated here, because overriding the `exclude` array is not possible
+                - org.springframework.boot.actuate.autoconfigure.metrics.data.RepositoryMetricsAutoConfiguration
+                - org.springframework.boot.actuate.autoconfigure.metrics.jdbc.DataSourcePoolMetricsAutoConfiguration
+                - org.springframework.boot.actuate.autoconfigure.metrics.startup.StartupTimeMetricsListenerAutoConfiguration
+                - org.springframework.boot.actuate.autoconfigure.metrics.task.TaskExecutorMetricsAutoConfiguration
+                - org.springframework.boot.actuate.autoconfigure.metrics.web.tomcat.TomcatMetricsAutoConfiguration
+
+Furthermore, you will need some configuration related to version control and continuous integration.
+
+3. Build agents require access to the VC server. Therefore, you need to add Artemis admin credentials so the build agent can access the repositories:
+
+.. code-block:: yaml
+
+    artemis:
+        version-control:
+            url: <url-to-your-vc-server>
+            default-branch: main # The branch that should be used as default branch for all newly created repositories. This does NOT have to be equal to the default branch of the VCS
+            # Artemis admin credentials
+            user: <artemis-admin>
+            password: <artemis-admin-password>
+
+4. Configuration related to the execution of build jobs:
+
+.. code-block:: yaml
+
+    artemis:
+        continuous-integration:
+            docker-connection-uri: unix:///var/run/docker.sock
+            specify-concurrent-builds: true                     # Set to false, if the number of concurrent build jobs will be chosen automatically based on system resources
+            concurrent-build-size: 1                            # If previous value is true: Set to desired value but keep available system resources in mind
```

Comment:
```suggestion
            concurrent-build-size: 1                            # If previous value is false: Set to desired value but keep available system resources in mind
```
If the previous value is `true`, this setting should be ignored since it is chosen automatically instead.

Reply:
It is actually the other way around. If `specify-concurrent-builds` is true, the value of `concurrent-build-size` is used. If false, the value is ignored automatically chosen. You can check `LocalCIConfiguration.java`, which uses the values, to verify

Reply:
Makes sense when re-reading the comments in the two lines again. :+1: